packages/lib/src/components/PromptPay/PromptPay.test.ts

```diff
@@ -0,0 +1,24 @@
+import PromptPay from './PromptPay';
+
+describe('formatProps', () => {});
```

Comment:
I meant about this describe block 'formatProps' - there is no point to have it in this test, it is not doing anything